os/arch/Kconfig

```diff
@@ -768,8 +768,7 @@ config FS_TMPFS_HEAP_INDEX
 	int "TMPFS Heap index"
 	default 1
 	depends on FS_TMPFS
-	depends on MM_NHEAPS > 1
-	depends on MM_REGIONS > 1
+	depends on != MM_NHEAPS
```

Comment:
Is it valid? `!=MM_NHEAPS`?
How about `MM_NHEAPS != 1` like others?

Reply:
sorry, I mistake modify MM_NHEAPS != 1